src/newfight.cpp

```diff
@@ -891,7 +891,7 @@ bool hit_with_multiweapon_toggle(struct char_data *attacker, struct char_data *v
     // Namely: We require that the attack's power is greater than double the spirit's force, otherwise it takes no damage.
     // If the attack's power is greater, subtract double the level from it.
     if ((IS_SPIRIT(def->ch) || IS_ANY_ELEMENTAL(def->ch))
-        && (!att->weapon || GET_WEAPON_FOCUS_RATING(att->weapon) == 0 || !WEAPON_FOCUS_USABLE_BY(att->weapon, att->ch)))
+        && ((!att->weapon && !GET_POWER(att->ch, ADEPT_KILLING_HANDS)) || GET_WEAPON_FOCUS_RATING(att->weapon) == 0 || !WEAPON_FOCUS_USABLE_BY(att->weapon, att->ch)))
```

Comment:
What is the intent of this change? I'm not quite understanding it.

Reply:
Generally, if a character is attempting to use unarmed or cyberweapon combat against spirits, immunity to normal weapons apply. But that's not the case if the character is using the killing hands power. 

Thus, if unarmed/cyberweapons and no killing hands, then proceed with immunity to normal weapons. Same applies if killing hands is active while using a weapon, since killing hands doesn't transfer to weapons.

Reply:
Got it, in that case the logic tree is wrong here. In the case of someone who has no weapon and has turned on killing hands, it'll then query the focus rating of their non-existent weapon and crash. Might need an inline conditional here, like:

```c++
if ((IS_SPIRIT(def->ch) || IS_ANY_ELEMENTAL(def->ch))
     && (att->weapon ? (GET_WEAPON_FOCUS_RATING(att->weapon) == 0 || !WEAPON_FOCUS_USABLE_BY(att->weapon, att->ch)) : !GET_POWER(att->ch, ADEPT_KILLING_HANDS))
```

Reply:
Ok, think I got it now. If the target is an elemental or spirit, and the attacker is not using killing hands while unarmed, and the attacker is not using a weapon focus that they are bonded to, then immunity to normal weapons applies.

Reply:
Yeah, I should read your comment all the way through >.>